app/src/main/java/com/infomaniak/mail/ui/main/folder/ThreadListViewModel.kt

```diff
@@ -58,4 +63,20 @@ class ThreadListViewModel @Inject constructor(
         // Delete Search data in case they couldn't be deleted at the end of the previous Search.
         searchUtils.deleteRealmSearchData()
     }
+
+    fun checkWebViewVersion(canShowWebViewOutdated: Boolean) {
+        isWebViewOutdated.value = if (canShowWebViewOutdated) {
+            WebViewCompat.getCurrentWebViewPackage(appContext)?.versionName?.let { versionName ->
+                val majorVersion = versionName.substring(0, versionName.indexOfFirst { it == '.' }).toInt()
+                // First known version of Android WebView System without bug
+                majorVersion < WEBVIEW_MIN_VERSION
+            } ?: false
+        } else {
+            false
+        }
```

Comment:
Shouldn't we handle the case where it can't be parsed as an Int so the app doesn't crash?